### PR TITLE
[8.0] ReplicateAndRegister  with FTS checks SE status 

### PR DESCRIPTION
**Pull request by chaen**
For some (certainly bad) reasons, the SE check was done only in the DMS replication. Do it always

BEGINRELEASENOTES
*DMS
NEW: ReplicateAndRegister FTS mode also checks the SE status
NEW: add a delay in monitoring in the FTS3Agent

ENDRELEASENOTES

### CODE REVIEW

**Review by chrisburr — DISMISSED** [review]

File: src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py [hunk 1]
```diff
@@ -184,6 +184,36 @@ def __call__(self):
         if self.rmsMonitoring:
             self.rmsMonitoringReporter = MonitoringReporter(monitoringType="RMSMonitoring")
 
+        # # source SE
```
Comment (andresailer) [inline src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py]:
```suggestion
```

File: src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py [hunk 2]
```diff
@@ -184,6 +184,36 @@ def __call__(self):
...
+        sourceSE = self.operation.SourceSE if self.operation.SourceSE else None
+        if sourceSE:
+            # # check source se for read
```
Comment (andresailer) [inline src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py]:
```suggestion
            # check sourceSE for read
```

File: src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py [hunk 3]
```diff
@@ -184,6 +184,36 @@ def __call__(self):
...
+                        self.rmsMonitoringReporter.addRecord(self.createRMSRecord(status, len(self.operation)))
+                    self.rmsMonitoringReporter.commit()
+                return bannedSource
+
+            if bannedSource["Value"]:
+                self.operation.Error = f"SourceSE {sourceSE} is banned for reading"
+                self.log.info(self.operation.Error)
+                return S_OK(self.operation.Error)
+
+        # # check targetSEs for write
```
Comment (andresailer) [inline src/DIRAC/DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py]:
```suggestion
        # check targetSEs for write
```

File: src/DIRAC/DataManagementSystem/Agent/FTS3Agent.py [hunk 4]
```diff
@@ -165,13 +169,19 @@ def getFTS3Context(self, username, group, ftsServer, threadID):
 
             log.debug(f"UserDN {userDN}")
 
+            # Chose a meaningfull proxy name for easier debugging
```
Comment (andresailer) [inline src/DIRAC/DataManagementSystem/Agent/FTS3Agent.py]:
```suggestion
            # Chose a meaningful proxy name for easier debugging
```

**Review by andresailer — COMMENTED** [review]
Just nitpicking.

Looks good to me.

**Review by fstagni — APPROVED** [review]

**Pull request merged**